.github/workflows/run-tests.yml

```
@@ -24,7 +24,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        go: [1.16.15, 1.17.13, 1.18.9, 1.19.4]
+        go: [1.18.9, 1.19.4]
```

shouldn't we consider the newest stable versions ?
```suggestion
        go: [1.21.13, 1.22.12, 1.23.6, 1.24.0]
```